### PR TITLE
Deprecate Previous Epoch Timeouts Count from Leader Index Computation

### PR DESCRIPTION
**Pull request by tholonious**

### CODE REVIEW

**Comment by tholonious** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1203?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1203** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1203?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1202** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1202?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1200** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1200?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1201** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1201?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1199** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/1199?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tholonious and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/pos_blockchain.go [hunk 1]
```diff
@@ -1133,12 +1133,12 @@ func (bav *UtxoView) hasValidProposerPartialSignaturePoS(block *MsgDeSoBlock, sn
 // block height + view number pair. It returns a bool indicating whether
 // we confirmed that the leader is valid. If we receive an error, we are unsure
 // if the leader is invalid or not, so we return false.
-func (bc *Blockchain) hasValidBlockProposerPoS(block *MsgDeSoBlock, parentUtxoView *UtxoView) (_isValidBlockProposer bool, _err error) {
-	currentEpochEntry, err := parentUtxoView.GetCurrentEpochEntry()
+func (utxoView *UtxoView) hasValidBlockProposerPoS(block *MsgDeSoBlock) (_isValidBlockProposer bool, _err error) {
```
Comment (lazynina) [inline lib/pos_blockchain.go:1136]: We generally name the receivers `bav`. Goland gets mad at me when the receiver names don't match.

**Review by lazynina — APPROVED** [review]
Just a nit on the receiver name

**Comment by tholonious** [comment]
### Merge activity

* **Apr 11, 2:26 PM EDT**: @tholonious started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/1203).

**Pull request merged**